index.html

```diff
@@ -0,0 +1,155 @@
+			<div class="pricing-detail">1: $1.99</div>
+			<div class="pricing-detail">2-50: $1.49</div>
+			<div class="pricing-detail">51+: 0.99</div>
+		</section>
+	</article>
+
+	<article class="product-card">
+		<header class="product-title">
+			<h1>Product Title</h1>
+		</header>
+		<section class="product-info">
+			<img src="img/logo.png" alt="Product image">
+			<div class="product-description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+			<div class="availability">Available</div>
+		</section>
+		<section class="product-specs">
+			<div class="product-specs-content">
+				<header class="spec-header">
+					Specifications
+				</header>
+				<div class="spec-detail">Size: 4" x 4"</div>
+				<div class="spec-detail">Weight: 8 oz.</div>
+			</div>
+			<footer>
+				<div class="spec-until">Specifications valid until 04/01/2016</div>
+			</footer>
+		</section>
+		<section class="product-pricing">
+			<header class="pricing-header">
+				Pricing
+			</header>
+			<div class="pricing-detail">1: $1.99</div>
+			<div class="pricing-detail">2-50: $1.49</div>
+			<div class="pricing-detail">51+: 0.99</div>
+		</section>
+	</article>
+
+	<article class="product-card">
+		<header class="product-title">
+			<h1>Product Title</h1>
+		</header>
+		<section class="product-info">
+			<img src="img/logo.png" alt="Product image">
+			<div class="product-description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+			<div class="availability">Available</div>
+		</section>
+		<section class="product-specs">
+			<div class="product-specs-content">
+				<header class="spec-header">
+					Specifications
+				</header>
+				<div class="spec-detail">Size: 4" x 4"</div>
+				<div class="spec-detail">Weight: 8 oz.</div>
+			</div>
+			<footer>
+				<div class="spec-until">Specifications valid until 04/01/2016</div>
+			</footer>
+		</section>
+		<section class="product-pricing">
+			<header class="pricing-header">
+				Pricing
+			</header>
+			<div class="pricing-detail">1: $1.99</div>
+			<div class="pricing-detail">2-50: $1.49</div>
+			<div class="pricing-detail">51+: 0.99</div>
+		</section>
+	</article>
+
+</body>
+</html>
```

Comment:
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior